### PR TITLE
Migrate to MUnit and introduce HelpMessage

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -32,15 +32,19 @@ inThisBuild(
...
     if (SettingKey[Boolean]("nativeLinkStubs").?.value.contains(true))
       List(
-        "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % Test,
+        "org.scalameta" %%% "munit" % "0.4.5",
         "com.github.lolgab" %%% "scalacheck" % "1.14.1" % Test
       )
     else
       List(
-        "org.scalatest" %%% "scalatest" % "3.0.8" % Test,
+        "org.scalameta" %%% "munit" % "0.4.5",
```
Comment (dwijnand) [inline build.sbt:47]: Was this meant to be in Compile scope?

Just noticed it's a transitive dependency of scalafix-core.

Comment (olafurpg) [reply build.sbt:47]: Whoop, that explains why MUnit has so many downloads 🙈 😓 😰 The Scalafix download numbers are only ~1.4k for scalafix-core_2.12:0.9.12+ but I see MUnit is also a transitive dependency of scalafmt-core, which has a lot more downloads 😅

Comment (olafurpg) [reply build.sbt:47]: Fixed in 	https://github.com/olafurpg/metaconfig/pull/86 I'll also correct my tweet.

Comment (olafurpg) [reply build.sbt:47]: It's maybe not so bad, the latest scalafmt versions only have ~5k unique IP downloads. The most downloaded versions don't depend on munit

Screenshot 2020-04-10 at 10 22 03
Screenshot 2020-04-10 at 10 15 31

**Pull request merged**